### ISSUE TITLE
When using system emoji, they are placed above the default ones in the page header.

### ISSUE THREAD

**Issue by Nek-12** [dragonwocky/notion-enhancer#23]
**describe the bug**
The images in page headers are placed above the default ones. In other places (e.g. page previews, list entries) they are replaced correctly though.
On the image below you can see the default emoji and then the Win-10 one drawn beneath it.

**expected behaviour**
Only the system version of the emoji should be displayed

**screenshots**
![image](https://user-images.githubusercontent.com/46318640/87655882-72ccc900-c761-11ea-9698-7ea1c2dd125f.png)

**platform**
Windows 10, native app.

**Comment by dragonwocky** [comment]
Add
```js
document.querySelectorAll('[src*="notion-emojis.s3-us-west-2.amazonaws.com"]:not(.notion-emoji)').forEach(el => el.outerHTML = '')
```
between lines 168 and 169.

Will push with the next update, ty for letting me know.

**Comment by dragonwocky** [comment]
fix released in v0.8.0
